README.md

```diff
@@ -1,52 +1,70 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
+# Getting Started with Create React App
```

Comment:
- [ ] Kindly update your README with Microverse template or similar. you can find it [here](https://github.com/microverseinc/readme-template).